CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Unreleased
 
+## 1.146.0 (October, 13 2023)
+ENHANCEMENTS:
+* resource/spotinst_ocean_aks_np: Added `accelerated_networking`, `disk_performance`, `min_gpu`, `max_gpu`, `min_nics`, `min_data` and `vm_types` fields in `vm_sizes` - `filters` object
+* resource/spotinst_ocean_aks_np_virtual_node_group: Added `accelerated_networking`, `disk_performance`, `min_gpu`, `max_gpu`, `min_nics`, `min_data` and `vm_types` fields in `vm_sizes` - `filters` object
+
```

Comment:
mentioning filters object is enough, no need to mention vm_sizes

Reply:
change